using-mobile-extensions/adobe-target/README.md

```diff
@@ -13,15 +13,16 @@ To get started with Target, follow these steps:
 
 ## Configure the Target extension in Experience Platform Launch      <a id="configuring-the-adobe-target-extension-in-adobe-launch"></a>
 
-![Adobe Target Extension Configuration](../../.gitbook/assets/screen-shot-2018-10-05-at-1.47.51-pm.png)
+![Adobe Target Extension Configuration](../../.gitbook/assets/adobe-target-launch-options.png)
 
 1. In Experience Platform Launch, click the **Extensions** tab.
 2. On the **Installed** tab, locate the Adobe Target extension, and click **Configure**.
 3. Your **Target** client code will be detected automatically.
 4. Optionally, provide your Environment ID.
 5. Set the timeout value to at least 5 seconds.
-6. Click **Save**.
-7. Follow the publishing process to update SDK configuration
+6. Optionally, enter the target workspace property token (Generated from Target UI).
```

Comment:
please capitalize "target" to "Target" and change the sentence to read as follows:

"Optionally, enter the Target workspace property token that was generated from Target UI."

Reply:
done